scholarly/publication_parser.py

```diff
@@ -320,8 +320,8 @@ def fill(self, publication: Publication)->Publication:
                     publication['bib']['abstract'] = result
                 elif key == 'total citations':
                     publication['cites_id'] = re.findall(
-                        _SCHOLARPUBRE, val.a['href'])[0]
-                    publication['citedby_url'] = _CITEDBYLINK.format(publication['cites_id'])
+                        _SCHOLARPUBRE, val.a['href'])[0].split(',')
+                    publication['citedby_url'] = _CITEDBYLINK.format(','.join(publication['cites_id']))
```

Comment:
Requiring `cites_id` to be a list as opposed to a single string of comma-separated values means that we have to do a `split` and `join` operation to construct the `citedby_url`. I'll respect the admins' decision to have `cites_id` as a list if preferable, but what ultimately matters is that the `citedby_url` is constructed properly and @mmontevil 's commit alone is sufficient to do that. Given that the sole purpose of `cites_id` in this library is to construct the `citedby_url`, leaving it as a single string seems preferable to me.